processor/groupbytraceprocessor/go.sum

```diff
@@ -48,25 +48,20 @@ github.com/Azure/go-autorest/logger v0.1.0/go.mod h1:oExouG+K6PryycPJfVSxi/koC6L
 github.com/Azure/go-autorest/logger v0.2.0/go.mod h1:T9E3cAhj2VqvPOtCYAvby9aBXkZmbF5NWuPV8+WeEW8=
```

Comment:
Remove go.sum from PR. We don't want this changing if no dependencies have changed just because of different developer setups.